pom.xml

```diff
@@ -64,19 +66,31 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
+			<exclusions><!--
 			<exclusions>
 				<exclusion>
 					<groupId>org.junit.vintage</groupId>
 					<artifactId>junit-vintage-engine</artifactId>
 				</exclusion>
+				-->
 			</exclusions>
 		</dependency>
 		<dependency>
 			<groupId>io.projectreactor</groupId>
 			<artifactId>reactor-test</artifactId>
 			<scope>test</scope>
 		</dependency>
-
+		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-api</artifactId>
+			<version>1.7.13</version>
```

Comment:
Wondering if you can indent the file. Some of the indentation seems to be off.

Reply:
It was properly indented.  I think that might have been a function of the web merge tool.

Reply:
can we delete the exclusion?

Reply:
Hopefully, it is fixed post rebase.